language/German/strings.po

```diff
@@ -1188,8 +1164,7 @@ msgid ""
 msgstr ""
 "\n"
 "\n"
-"Drücken Sie OK oder bewegen Sie den Joystick nach links und dann nach rechts "
-"um fortzufahren"
+"Drücken Sie OK oder bewegen Sie den Joystick nach links und dann nach rechts, um fortzufahren"
```

Comment:
can you provide a better translation? I did a quick translate with google and the above German is the same, but when re-translated back to English it comes out as either "Tip OK" or "Enter Move OK"
do you have a word for "Type"? I see "Typing" is "Maschinenschreiben" but "Type" comes back in German as "Art"
?
If you can provide a better German translation please let us know. Thanks